devutils/git-hooks/pre-commit

```diff
@@ -13,9 +13,16 @@ source "$(dirname "${BASH_SOURCE[0]}")/../lib/bash/prompt.bash"
 
 # Rules checking Go code validity
 PreCommit::GoChecks () {
+    local filepath="$1"
+
     [ -z "$GOPATH" ] && Log::Failure "please install golang on workstation (\$GOPATH missing)"
+    local golint="$GOPATH/bin/golint"
+    [ -x "$golint" ] || Log::Failure "golint not found. Please run 'go get -u github.com/golang/lint/golint'."
```

Comment:
1) [Golint](https://github.com/golang/lint) is a tool that should be installed through the go utility (`go get github.com/golang/lint`). If `$GOPATH/bin/golint` is not executable but exists, it means user screwed up its `$GOPATH` directory which is not likely to happen.